:postbox:  2026 roadmap (#6347)
actualbudget/actual: packages/docs/blog/2025-12-20-roadmap-for-2026.md

---
title: Roadmap for 2026

> description: We were able to accomplish a lot in 2025. Let's look back over the year and also look at whats to come.

Reviewer: ```suggestion
description: We were able to accomplish a lot in 2025. Let's look back over the year and also look at what's to come.
```

- Multiple new report types
- NPM package for the sync server
- Translations
- Bank sync management page
- Desktop app improvements such as the embedded server
- Auto-hold income
- Merge transactions
- Transaction tags
- Mobile pages for Rules, Payees, Schedules, and Bank Sync
- Budget Currencies (still experimental)
- Quickly add single schedules from the account view

**Note on Translations:**  Translation in a never ending project.
Every month there are new or modified strings that need translated.
We could use as many contributors to translation as we can get.
If you are interested in helping translate Actual into other languages see [our translation guide].

If you remember our roadmap from last year, you will see that we were able to accomplish all but one of our goals, plugins.
I will touch on that more below.

### Maintainer and contributor payments
This year we started paying small stipends to the core maintainer team, and giving out SWAG items to other contributors.
The maintainer stipends are based on points received for doing the necessary, but not very exciting parts, of maintaining Actual
This is only possible because of the generous donations from you the users.

> Many of you donate regularly and there are over 8000 Pikapods users who's hosting fees partially go towards supporting Actual.

Reviewer: ```suggestion
Many of you donate regularly and there are over 8000 Pikapods users whose hosting fees partially go towards supporting Actual.
```

We offer a very hearty thank you to those who are donating.
See the bottom of the post for ways to support the project if you would like to donate.

## 2026 Wish list

**Plugins**: We have been working on adding plugins to allow users to more easily add functionality to the app without the need to add said feature to the base app.
The feature is not quite ready as plugins have a tendency to break existing features, which slows down development a great deal.
We are doing everything we can to not break existing features that we know you rely on.

> Our hope is to have the feature ready for you develop for in the first few months of 2026.

Reviewer: ```suggestion
Our hope is to have plugins ready for you to build on in the first few months of 2026.
```
